### PR TITLE
Release 4.15.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 4.15.0 (2020-06-15)
+
+- Updated the lookup component to include a button which triggers the show more picker from the input box. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Updated the lookup component to include a search button in the input field that opens the show more picker. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)
```
"search button" and "input field" are the terminology in the design guidelines. If we actually mean the

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.15.0](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.15.0)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- Updated the lookup component's dropdown keyboard interactions. [#176](https://github.com/blackbaud/skyux-lookup/pull/176)
+- Fixed the lookup component to refocus the input box after closing the show more picker. [#193](https://github.com/blackbaud/skyux-lookup/pull/193)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:5]:
```suggestion
- Fixed the lookup component to focus on the input field after the show more picker closes. [#193](https://github.com/blackbaud/skyux-lookup/pull/193)
```
Same note as before that if we actually mean the input box component and not the lookup component's input field, we probably need different edits here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# 4.15.0 (2020-06-15)
+
+- Updated the lookup component to include a button which triggers the show more picker from the input box. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)
+- Updated the lookup component's dropdown keyboard interactions. [#176](https://github.com/blackbaud/skyux-lookup/pull/176)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:4]:
```suggestion
- Updated the keyboard interactions for the lookup component's dropdown menu. [#176](https://github.com/blackbaud/skyux-lookup/pull/176)
```

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.15.0](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.15.0)

**Pull request merged**